src/components/WalletSection.tsx

```diff
@@ -1,29 +1,22 @@
-import PropTypes from 'prop-types';
-import React from 'react';
+import React, {ReactNode} from 'react';
 import useThemeStyles from '@styles/useThemeStyles';
 import Section from './Section';
 
-const propTypes = {
+type WalletSectionProps = {
     /** Contents to display inside the section */
-    children: PropTypes.node,
+    children?: ReactNode;
 
     /** The icon to display along with the title */
-    icon: PropTypes.func,
+    icon?: (props: unknown) => React.ReactNode;
 
     /** The text to display in the subtitle of the section */
-    subtitle: PropTypes.string,
+    subtitle?: string;
 
     /** The text to display in the title of the section */
-    title: PropTypes.string.isRequired,
+    title: string;
 };
 
-const defaultProps = {
-    children: null,
-    icon: null,
-    subtitle: null,
-};
-
-function WalletSection({children, icon, subtitle, title}) {
+function WalletSection({children, icon, subtitle, title}: WalletSectionProps) {
```

Comment:
please add default values to minimize chances of any regressions 😄

Reply:
@kubabutkiewicz We don't need in this case I believe, if not defined they will be `undefined` so it's basically the same.